_includes/default-footer.html

```diff
@@ -41,9 +41,7 @@ <h3>Get in touch</h3>
             The code behind the infrastructure is licensed under the <a href="{{ site.github.repository_url }}/blob/main/LICENSE.md">MIT License</a>
           </p>
           <p>
-            The logo was originally designed by
-            <a href="https://github.com/lukethebaka">Lucas Emanoell Medeiros e Silva</a> and
-            then adapted by the team.
+            Logo design by [Jilaga Nneoma]({% link people.md %}#jilaga). Other OLS branding assets are found [here]({% link branding.md %}).
```

Comment:
It looks like the footer.html file doesn't process the markdown into HTML, so we have to use html for the link instead. I've also changed the link name from "here" to something more descriptive... (see why: https://www.smashingmagazine.com/2012/06/links-should-never-say-click-here/ ) 

```suggestion
            Logo design by <a href="{{ link people.md }}#jilaga">Jilaga Nneoma</a>. Other OLS branding assets are found at the <a href="{% link branding.md %}">OLS branding page</a>.
```

Reply:
This is **very** useful. Thank you!